canonicalwebteam/templates/image_template.html

```diff
@@ -6,7 +6,7 @@
   alt="{{ alt }}"
   width="{{ width }}"
   height="{{ height }}"
-  class="lazyload{% if extra_classes %} {{ extra_classes }}{% endif %}"
+  class="{% if lazy != False %}lazyload{% endif %}{% if extra_classes %}{{ extra_classes }}{% endif %}"
```

Comment:
you also need to change the attributes from `data-src` and `data-srcset` to `src` and `srcset` if `lazy == False` on lines 3 and 5.